### PR TITLE
[TC-46] - Update TO client for psql API changes.

### PR DESCRIPTION
**Pull request by dneuman64**
Updated the TO client to work with API changes with psql.  This fixes [TC-46](https://issues.apache.org/jira/browse/TC-46)

### CODE REVIEW

File: traffic_ops/client/delivery_service_resources.go [hunk 1]
```diff
@@ -35,49 +34,49 @@ type DeleteDeliveryServiceResponse struct {
...
+	RangeRequestHandling int                    `json:"rangeRequestHandling"`
 	HeaderRewrite        string                 `json:"headerRewrite"`
 	EdgeHeaderRewrite    string                 `json:"edgeHeaderRewrite"`
 	MidHeaderRewrite     string                 `json:"midHeaderRewrite"`
 	TRResponseHeaders    string                 `json:"trResponseHeaders"`
 	RegexRemap           string                 `json:"regexRemap"`
 	CacheURL             string                 `json:"cacheurl"`
 	RemapText            string                 `json:"remapText"`
-	MultiSiteOrigin      string                 `json:"multiSiteOrigin"`
+	MultiSiteOrigin      int                    `json:"multiSiteOrigin"`
```
Comment (mitchell852) [inline traffic_ops/client/delivery_service_resources.go]: this should be a bool

File: traffic_ops/client/fixtures/delivery_service.go [hunk 2]
```diff
@@ -22,30 +22,30 @@ func DeliveryServices() *client.GetDeliveryServiceResponse {
...
+				MaxDNSAnswers:        0,
+				MissLat:              44.654321,
+				MissLong:             -99.123456,
+				Protocol:             0,
 				IPV6RoutingEnabled:   true,
-				RangeRequestHandling: "0",
+				RangeRequestHandling: 0,
 				TRResponseHeaders:    "Access-Control-Allow-Origin: *",
-				MultiSiteOrigin:      "0",
+				MultiSiteOrigin:      0,
```
Comment (mitchell852) [inline traffic_ops/client/fixtures/delivery_service.go]: same comment as above. this is a bool in the database.

File: traffic_ops/client/delivery_service_resources.go (hunk 1 above)
Comment (dneuman64) [inline traffic_ops/client/delivery_service_resources.go]: good catch, will fix

**Comment by dneuman64** [comment]
thanks @mitchell852!

**Pull request merged**